scripts/dashboard.coffee

```diff
@@ -49,6 +63,7 @@ module.exports = (robot) ->
                 Dashboard URL manager
                 `!dashboard set <url>` : Switch to given URL
                 `!dashboard sumo <dashboardId>` : Switch to the dashboardId URL
+                `!dashboard show <dashboardId>-<widgetId>` : Get picture of a widget
```

Comment:
You should buy one :stuck_out_tongue_winking_eye: 
Really under construction lol, I think I will move my dev in an other place at the end